presto-common/src/main/java/com/facebook/presto/common/type/StandardTypes.java

```diff
@@ -45,6 +45,8 @@
     public static final String IPPREFIX = "ipprefix";
     public static final String GEOMETRY = "Geometry";
     public static final String BING_TILE = "BingTile";
+    public static final String BIGINT_ENUM = "BigintEnum";
+    public static final String VARCHAR_ENUM = "VarcharEnum";
```

Comment:
Is this user visible? The names are a bit odd. Not sure if  enum<varchar> will work better

Reply:
There are 2 types of enums and they cannot be both of type "enum". This would be visible in error messages and plans. If you have better suggestions I'm all ears.